_config/documentconverter.yml

```diff
@@ -4,11 +4,6 @@ Only:
   moduleexists: silverstripe/documentconverter
   EnvVarSet: DOCVERT_USERNAME
 ---
-SilverStripe\DocumentConverter\ServiceConnector:
-  username: '`DOCVERT_USERNAME`'
-  password: '`DOCVERT_PASSWORD`'
-  url: '`DOCVERT_URL`'
-
```

Comment:
Ok, so after seeing the response on https://github.com/silverstripe/silverstripe-framework/issues/7987 it seems that this isn't supposed to work. In that case removing it is fine, but we still need to replace it with something.

I suggest perhaps using \_config.php and the `ServiceConnector::setUsername` etc APIs inside an `Environment::getEnv` check

Reply:
The module reads from the environment if no config is set. I'm not sure that's necessary at all, as there should be no constants `define`d for SS4 based sites.

Reply:
Ok, great